tests/sentry/incidents/endpoints/validators/test_validators.py

```diff
@@ -814,6 +814,48 @@ def test_update_anomaly_detection_snuba_query(
             event=audit_log.get_event_id("DETECTOR_EDIT"),
             data=dynamic_detector.get_audit_log_data(),
         )
+        mock_audit.reset_mock()
```

Comment:
i'd recommend splitting this out into a new test case or breaking down this larger test.

Reply:
broke them out 👍